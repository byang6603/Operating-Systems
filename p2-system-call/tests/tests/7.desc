Fork Test: Parent forks, child writes crash string and exits. Parent calls crash. Should not panic.
