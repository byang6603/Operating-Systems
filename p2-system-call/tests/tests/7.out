XV6_TEST_OUTPUT Parent calling crash!
XV6_TEST_OUTPUT Parent exiting!!
